docs/source/features/stablehlo.md

```diff
@@ -64,6 +64,31 @@ print(stablehlo.mlir_module())
 The second to last line we used `jax.ShapedDtypeStruct` to specify the input shape.
 You can also pass a numpy array here.
 
+### Inline some weights in generated stablehlo
+
+Suppose that you want to inline some (or all) of the model's weight
```

Comment:
Suggest: 
You can inline some or all of your model's weights into the StableHLO graph as  constants by exporting a separate function that calls your model.

Reply:
done.